webapp/channels/src/components/suggestion/search_channel_with_permissions_provider.test.tsx

```diff
@@ -278,4 +277,4 @@ describe('components/SearchChannelWithPermissionsProvider', () => {
 
         expect(args.items.length).toEqual(0);
     });
-});
+});
```

Comment:
style checker will complain because there is no new line

Reply:
done